bundles/io/org.eclipse.smarthome.io.json/OSGI-INF/org.eclipse.smarthome.io.json.internal.gson.OsgiGsonBindingService.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file and put a `.gitignore` file in the `OSGI-INF` directory which ignores `/*.xml` files so auto-generated files like this do not end up in the repository.

Reply:
Done